tests/unit/serializeLikeHarvester.test.ts

```diff
@@ -403,9 +403,7 @@ describe('serializeLikeHarvester', () => {
       call_args: [
         {
           name: 'who',
-          value: {
```

Comment:
@prashantasdeveloper I am not sure why this changed.

I am assuming the shorter version is correct, but I don't get how the test suite got out of sync here